src/sentry/dynamic_sampling/feature_multiplexer.py

```diff
@@ -54,5 +58,12 @@ def get_user_biases(user_set_biases):
         return returned_biases
 
     @staticmethod
-    def get_supported_biases_ids():
+    def get_supported_biases_ids() -> Set[str]:
         return {bias["id"] for bias in DEFAULT_BIASES}
+
+    @classmethod
+    def get_user_bias(cls, bias_id: str, user_set_biases: Optional[List[Bias]]) -> Bias:
```

Comment:
Lets please rename this to something else 
It is very confusing with `get_user_biases`

Reply:
@ahmedetefy wdyt?
```
get_user_bias_by_id
```
?

Reply:
Yes that works

Reply:
fixed via a170022aae64f724d1057a16d7a53de56b650a67